utils/api.py

```diff
@@ -898,6 +900,35 @@ def get_pdf_report(task_id):
     return response
 
 
+@app.route('/api/v1/tasks/<task_id>/stix2', methods=['GET'])
+def get_stix2_bundle_from_report(task_id):
+    '''
+    Generates a STIX2 Bundle with indicators generated of a JSON report.
+    '''
+    formatting = request.args.get('pretty', default='False', type=str)[0].lower()
+
+    report_dict, success = get_report_dict(task_id)
+
+    if formatting == 't' or formatting == 'y' or formatting == '1':
```

Comment:
You could move the formatting stuff below line 918, so if get_report_dict is not successful, it returns more quickly.